google/generativeai/client.py

```diff
@@ -59,6 +59,8 @@ def create_file(
         mime_type: str | None = None,
         name: str | None = None,
         display_name: str | None = None,
+        resumable: bool | None = None,
```

Comment:
Are we exposing too many details here?

Do we need the `resumable` arg, could it just be `resumable=chunk_size is not None`?

Is there a reasonable default policy we could implement like "if the file is over (2x)MB use resumable with chunk size (x)MB"?

Reply:
Yeah, fair point. After some exploration, how about let's start simple and just allow users to set resumable as a boolean flag? The chunk size needs to be a multiple of a certain size and is unfriendly for developers.